tools/wireshark/Makefile

```diff
@@ -0,0 +1,33 @@
+PLUGIN_NAME = retis
+
+SRC_FILES := $(wildcard *.c)
+HEADERS := $(wildcard *.h)
+OUTFILE = $(PLUGIN_NAME).so
+
+CC = gcc
+CFLAGS = -Wall -fPIC -I. $(shell pkg-config --cflags wireshark json-c)
+LDFLAGS = -shared $(shell pkg-config --libs wireshark json-c)
+
+WIRESHARK_VERSION_FULL := $(shell pkg-config --modversion wireshark json-c)
+WIRESHARK_VERSION := $(shell echo $(WIRESHARK_VERSION_FULL) | cut -d. -f1,2)
+
+WIRESHARK_PLUGIN_BASE_DIR ?= $(HOME)/.local/lib/wireshark/plugins
+WIRESHARK_PLUGIN_DIR := $(WIRESHARK_PLUGIN_BASE_DIR)/$(WIRESHARK_VERSION)/epan
+
+.PHONY: all clean check install
+
+all: check $(OUTFILE)
+
+check:
+	@pkg-config --exists wireshark || (echo "Wireshark development headers not found. Please install them (e.g., 'libwireshark-devel')." && false)
```

Comment:
s/libwireshark-devel/wireshark-devel/ ?

Reply:
Hehe, I merged "libwireshark-dev" (debian/ubuntu) with "wireshark-devel" (fedora) :-) I'll fix it